Submissions/engrshege.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "[Oluwasegun Idowu]",
+  "img": "[https://avatars.githubusercontent.com/u/52855925?v=4]",
+  "email": "[engrshege@gmail.com]",
+  "links": {
+    "website": "[https://engrshege.github.io/myportfolio/]",
```

Comment:
none of these should have brackets around the URLs

Reply:
I have removed the brackets from here too ma'am.

Thank you ma